### PR TITLE
Task/http client integration

### PR DESCRIPTION
**Pull request by leew**
Implemented the Http Client ready for integration with the front end.

### CODE REVIEW

File: modules/end-user-portal/src/Models/Response/Organization.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
+using Newtonsoft.Json;
+
+namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Models.Response;
+
+public class Organization
```
Comment (ChristopherJamesMorris) [inline modules/end-user-portal/src/Models/Response/Organization.cs:5]: Can we drop the American spelling and change it to Organisation please.

File: modules/end-user-portal/src/Models/Response/Organization.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public string Id { get; set; }
+    public string Name { get; set; }
+
+    public OrganizationAddress Address { get; set; }
+}
+
+// {
+//     "id": "FKG31",
```
Comment (ChristopherJamesMorris) [inline modules/end-user-portal/src/Models/Response/Organization.cs:25]: Can we remove the comments please.

File: modules/end-user-portal/src/Models/Response/Organization.cs (hunk 1 above)
Comment (leew) [reply modules/end-user-portal/src/Models/Response/Organization.cs:5]: Sure, we were matching the FHIR API spellings

**Comment by ChristopherJamesMorris** [comment]
These changes have been pulled into another branch so closing this PR.

**Pull request closed without merging**